### PR TITLE
Load Daniel font before rendering, provide default styling using inline css

### PR DESCRIPTION
**Pull request by StefRave**
right after the theme is loaded and before diagram.drawSvg
- Added support in snap theme to specify div name (compatible to Raphaël theme)
- Fix Raphaël and Snap detection
- Fixed unit test run

### CODE REVIEW

File: fonts/daniel/daniel_700.font.js [hunk 1]
```diff
@@ -23,6 +23,7 @@
  * License information:
  * http://creativecommons.org/licenses/by-nd/3.0/
  */
+if (typeof Raphael !== "undefined") {
```
Comment (bramp) [inline fonts/daniel/daniel_700.font.js]: Thanks, I was being lazy :)

File: test/test.html [hunk 2]
```diff
@@ -50,7 +50,7 @@
     	</textarea>
     	<button id="parse" type="button">Click Me!</button>  
 
-    	<svg id="diagram"></svg>
+    	<div id="diagram"></svg>
```
Comment (bramp) [inline test/test.html]: Opening div tag, but closing svg tag...

**Comment by StefRave** [comment]
I thought my in my first commit, the rendering of SVG was performed when the font was loaded. Only I forgot that I added the Font to the windows font store. This caused the Daniel font to load quickly on my PC.
The current solution adds an invisible div tag to the DOM. On the window.load event, the font will have been loaded. This will probably also mean that WebFontLoader is probably not needed anymore. I will check that some other time.

The current version uses an embedded stylesheet, which is possible to be overridden in the web page. What do you think of this solution?

The current solution still needs:    ``` <style type="text/css">@font-face { font-family: 'Daniel'; font-style: normal; font-weight: normal; src: local('Daniel'), url('../fonts/daniel/daniel.otf') format('opentype'); } </style>``` in the html page.
* We could embed this also in the embedded SGV stylesheet, but it will be a problem with the url.
* We could embed the complete font in base64 in the embedded stylesheet (like http://blog.patdavid.net/2012/08/embedding-fonts-with-css-and-base64.html). This is the only solution which completely backwards compatible, but will blow up the SVG. However, the Raphaël SVG "hand" files where also huge, because each letter was rendered using path.

What do you think is the way to go?

**Comment by StefRave** [comment]
Closed, because font rendering is supported in version 2

**Pull request closed without merging**